index.html

```diff
@@ -77,6 +122,47 @@
 	</div>
 
 	<script>
+
+		function getAllCookies(){
+		  var pairs = document.cookie.split(";");
```

Comment:
`var` has been deprecated for `let` and `const`

`var` is a global variable
`let` is a local variable
`const` is a local variable that can't be reassigned

please replace all `var`
`const` is preferred, where possible

Reply:
updated